plugins/flutter_aepedge/README.md

```diff
@@ -0,0 +1,183 @@
+
+Install instructions for this package can be found [here](https://pub.dev/packages/flutter_aepedge/install).
+
+> Note: After you have installed the SDK, don't forget to run `pod install` in your `ios` directory to link the libraries to your Xcode project.
+
+## Tests
+
+Run:
+
+```bash
+flutter test
+```
+
+## Usage
+### Edge Network
+
+For more detailed information on the Edge APIs, visit the documentation [here](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-edge)
+
+##### Registering the extension with AEPCore:
+
+ > Note: It is required to initialize the SDK via native code inside your AppDelegate (iOS) and MainApplication class (Android).
+
+As part of the initialization code, make sure that you set the SDK wrapper type to `Flutter` before you start the SDK.
+
+Refer to the [Initialization](https://github.com/adobe/aepsdk_flutter#initializing) section of the root README for more information about initializing the SDK.
+
+**Initialization Example**
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore setWrapperType:AEPWrapperTypeFlutter];
+
+     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    NSString* ENVIRONMENT_FILE_ID = @"YOUR-APP-ID";
+    
+    const UIApplicationState appState = application.applicationState;
+
+    NSArray *extensionsToRegister = @[AEPMobileEdgeIdentity.class, 
+                                      AEPMobileEdge.class,                                              
+                                      ];
+
+    [AEPMobileCore registerExtensions:extensionsToRegister completion:^{
+    [AEPMobileCore configureWithAppId: ENVIRONMENT_FILE_ID];
+    }];
+    return YES;   
+ } 
+```
+
+Android
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;  
+...
+import io.flutter.app.FlutterApplication;
+...
+public class MainApplication extends FlutterApplication {
+  ...
+  // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+  private final String ENVIRONMENT_FILE_ID = "YOUR-APP-ID";
+
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setWrapperType(WrapperType.FLUTTER);
+
+    Edge.registerExtension();
+    Identity.registerExtension();
+    MobileCore.start(new AdobeCallback () {
+        @Override
+        public void call(Object o) {
+          MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+        }
+   });
+```
+------
+#### Importing the SDK:
+```dart
+import 'package:flutter_aepedge/flutter_aepedge.dart';
+```
+------
+#### Getting edge version:
```

Comment:
Would be nice to use a similar format as RN here
```
## API reference
### extensionVersion
**Syntax**
\```dart
syntax example
\```

**Example**
\```dart
usage example
\```

### sendEvent
...
```